### PR TITLE
Release 2.2.0

### PR DESCRIPTION
**Pull request by florianb**

### CODE REVIEW

**Comment by florianb** [comment]
Ready to :shipit:

File: .editorconfig [hunk 1]
```diff
@@ -1,4 +1,4 @@
-root = true
+#root = true
```
Comment (sindresorhus) [inline .editorconfig]: ?

**Comment by sindresorhus** [comment]
👍 Nice work!

File: .editorconfig (hunk 1 above)
Comment (florianb) [inline .editorconfig]: Uhh, i commented it out to be able setting the indent_size to 3 (by having a higher order editorconfig), which is what i prefer. I will revert it if you don't agree with it.

**Comment by florianb** [comment]
Thanks - i appreciate it! 😄

**Pull request merged**